packages/client/src/app.tsx

```diff
@@ -1,44 +1,18 @@
 import { ToasterComponent, ToasterProvider, ThemeProvider } from '@gravity-ui/uikit'
-import React, { useState, useEffect, useMemo } from 'react'
-import { useSelector } from 'react-redux'
+import React from 'react'
 import { RouterProvider } from 'react-router-dom'
-import PageLoader from './components/pageLoader'
-import AuthContext from './context/authContext'
-import { selectUserLoading } from './reducers/user'
+import withAuthInfo from 'hoc/withAuthInfo'
 import router from './router'
-import AuthService from './service/auth.service'
 
-const App = () => {
-  const loading = useSelector(selectUserLoading)
-  const [isAuth, setAuth] = useState<boolean>(false)
-  const authInfo = useMemo(() => ({ isAuth, setAuth }), [isAuth])
+const App = () => (
+  <React.StrictMode>
+    <ThemeProvider theme="light">
+      <ToasterProvider>
+        <RouterProvider router={router} />
+        <ToasterComponent className="optional additional classes" />
```

Comment:
Можно было попутно удалить, это же явно лишняя копипаста из примера gravity-ui
```suggestion
        <ToasterComponent />
```